feg/gateway/services/n7_n40_proxy/n7/n7_create_test.go

```diff
@@ -0,0 +1,233 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package n7_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	n7_sbi "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControl"
+	sbi "magma/feg/gateway/sbi/specs/TS29571CommonData"
+	"magma/feg/gateway/services/n7_n40_proxy/n7"
+	"magma/lte/cloud/go/protos"
+)
+
+const (
+	IMSI1          = "123456789012345"
+	UE_IPV4        = "10.1.2.3"
+	PDU_SESSION_ID = 10
+	GPSI1          = "9876543210"
+	APN1           = "apn.magma.com"
+)
+
+var (
+	AccessType3gpp     = sbi.AccessType("3GPP_ACCESS")
+	DnnMagma           = sbi.Dnn(APN1)
+	Gpsi1              = sbi.Gpsi(GPSI1)
+	UeIpv4             = sbi.Ipv4Addr(UE_IPV4)
+	PduSessionTypeIpv4 = sbi.PduSessionType("IPV4")
+	RatTypeNR          = sbi.RatType("NR")
+	UeTzIST            = sbi.TimeZone("+05:30")
+	SmPolicyUrl        = "https://example.com//npcf-smpolicycontrol/v1/sm-policies/12345"
+	ActTime            = time.Unix(1634906551, 0)
+	DeactTime          = time.Unix(1634913751, 0)
+)
+
```

Comment:
good. The reason to have those variables here is that it is easy to change them in case you need them. And if test fails you can easily see values here